src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/HealthCertifiedPersonViewController.swift

```diff
@@ -99,6 +99,22 @@ class HealthCertifiedPersonViewController: UIViewController, UITableViewDataSour
 		}
 	}
 
+	func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+		if HealthCertifiedPersonViewModel.TableViewSection.map(section) == .certificates {
+			let footerView = UIView()
+			footerView.backgroundColor = .clear
+
+			return footerView
+		} else {
+			return UIView()
```

Comment:
Should this not be `nil` if there is no footer view?

Reply:
Yes 👍